JavaOpdracht6/familyTree/src/main/java/Person.java

```diff
@@ -115,29 +115,40 @@ public void setPets(List<Pet> pets) {
     }
 
     // Methods
-    public void addParents(Person mother, Person father) {
-        setMother(mother);
-        setFather(father);
+    public void addParents(Person mother, Person father, Person child) {
+        child.setMother(mother);
+        child.setFather(father);
+
+        addChild(mother, child);
+        addChild(father, child);
     }
 
     public void addChild(Person parent, Person child) {
         List<Person> kids = new ArrayList<>();
-        kids.add(child);
-        children = kids;
+        if (parent.getChildren() != null) {
+            // For-loop zonder index. 'c' is de naam van de variabele binnen de for-lus
+            for (Person c : parent.getChildren()) {
+                kids.add(c);
+            }
+        } else {
+            kids.add(child);
+        }
         parent.setChildren(kids);
     }
```

Comment:
List kids is niet nodig, this.children is List; 
Deze methode zet nieuwe child niet bij als parent.getChildren is not null
In  zou hier alleen this.children.add(child) zetten